dashboard/app/views/devise/shared/_oauth_links.haml

```diff
@@ -27,3 +27,8 @@
 :javascript
 
   $('.oauth_sign_in').click(dashboard.clientState.reset);
+  // If the makerBridge is available, we are in the Maker App, which has it's
+  // own Google login button. Hiding this google-oauth button to avoid confusion.
+  if (!!window.MakerBridge) {
```

Comment:
This is the `isCodeOrgBrowser` function, but I'm not sure how to use a function like that, which I would normally import into a js file. Any ideas?

Reply:
The proper architecture would probably be to take all of this file's JS out of the HAML file and have it live in `/apps` instead.